### PR TITLE
test(allocation): test 10k allocations on test networks

### PR DESCRIPTION
**Pull request by Ramarti**
In preparation for mainnet allocation, we test allocating tokens to 10k addresses at genesis.

issue: none

### CODE REVIEW

**Review by jdubpark — APPROVED** [review]

File: contracts/script/GenerateAlloc.s.sol [hunk 1]
```diff
@@ -360,6 +360,7 @@ contract GenerateAlloc is Script {
             vm.deal(0x13919a0d8603c35DAC923f92D7E4e1D55e993898, 100000000 ether);
         } else if (block.chainid == 1516) {
             // Odyssey testnet alloc
+            setTestAllocations();
```
Comment (limengformal) [inline contracts/script/GenerateAlloc.s.sol:363]: Can we make it optional?

Comment (Ramarti) [reply contracts/script/GenerateAlloc.s.sol]: Done

**Review by limengformal — APPROVED** [review]

File: contracts/script/GenerateAlloc.s.sol [hunk 2]
```diff
@@ -44,7 +44,8 @@ contract GenerateAlloc is Script {
 
     string internal dumpPath = getDumpPath();
     bool public saveState = true;
-    uint256 public constant MAINNET_CHAIN_ID = 0; // TBD
+    uint256 public constant MAINNET_CHAIN_ID = 1514; // TBD
+    bool private constant ALLOCATE_10K_TEST_ACCOUNTS = false;
```
Comment (limengformal) [inline contracts/script/GenerateAlloc.s.sol:48]: Add a comment so users know to change the value to perform the test.

Comment (Ramarti) [reply contracts/script/GenerateAlloc.s.sol:51]: Added

File: contracts/script/GenerateAlloc.s.sol (hunk 1 above)
Comment (Ramarti) [reply contracts/script/GenerateAlloc.s.sol]: Yes

**Pull request merged**